wayang-docs/src/main/resources/how_contribute/code_changes/preparing_contribute_code_changes/branching_strategy.md

```diff
@@ -0,0 +1,58 @@
+---
+license: |
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+    
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+layout: default
+title: "Branching Strategy"
+previous:
+    url: /how_contribute/code_changes/code_review_criteria/
+    title: Code Review Criteria
+next:
+    url: /how_contribute/code_changes/preparing_contribute_code_changes/jira_issue/
+    title: Jira issues
+---
+# Apache Wayang-Branching Strategy
+This page is a summary from an email thread: https://lists.apache.org/thread/v9nhw3ml585456byd6t6cbn3vbhnzkww
+
+## Branches:
+__*release (formerly main):*__ 
```

Comment:
We changed a bit to simplify the process and we will not have a branch called "release" for now. Only the rel/x.x.x